### PR TITLE
ipykernel 6.15.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: **no** breaking changes https://github.com/ipython/ipykernel/blob/v6.9.1/CHANGELOG.md
License: https://github.com/ipython/ipykernel/blob/v6.15.1/COPYING.md
Requirements: https://github.com/ipython/ipykernel/blob/v6.15.1/pyproject.toml

Actions:
1. Update scripts, host, run, and test dependencies and pinnings

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
CI issues on Prefect for `linux` platforms. The test_shutdown_subprocesses is failing has to do with shutting down the system using process groups. We put the entire build into a process group. It looks that the test is checking to see that a child process or process group is shutdown after it’s requested to, but it’s not.
We should skip `test_shutdown_subprocesses`.

File: recipe/run_test.py [hunk 1]
```diff
@@ -53,11 +56,17 @@ def go():
...
+    # with shutting down the system using process groups. 
+    # We put the entire build into a process group.
+    # It looks that the test is checking to see 
+    # that a child process or process group is shutdown after it’s requested to, but it’s not.
+    skips = ["flaky", "test_shutdown_subprocesses"]
 
     if len(skips) == 1:
         pytest_args += ["-k", "not {}".format(*skips)]
-    else:
+    else:
```
Comment (danpetry) [inline recipe/run_test.py]: trailing whitespace

File: recipe/run_test.py [hunk 2]
```diff
@@ -53,11 +56,17 @@ def go():
...
     ]
 
-    skips = ["flaky"]
+    # 2022/8/29: CI issues on Prefect for linux platforms:
+    # The test_shutdown_subprocesses is failing has to do 
+    # with shutting down the system using process groups. 
+    # We put the entire build into a process group.
+    # It looks that the test is checking to see 
+    # that a child process or process group is shutdown after it’s requested to, but it’s not.
+    skips = ["flaky", "test_shutdown_subprocesses"]
```
Comment (danpetry) [inline recipe/run_test.py]: Should `"test_shutdown_subprocesses"` be only skipped `if platform.system() == "Linux"`?

File: recipe/run_test.py (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/run_test.py]: Fixed

File: recipe/run_test.py (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/run_test.py]: Thanks. I've added skipping for `linux` platforms

**Comment by skupr-anaconda** [comment]
I opened a new PR for v6.15.2 https://github.com/AnacondaRecipes/ipykernel-feedstock/pull/23

**Pull request closed without merging**